ai-catalog/samples/gemini-video-summarization/src/main/java/com/android/ai/samples/geminivideosummary/VideoSummarizationScreen.kt

```diff
@@ -71,18 +74,21 @@ import java.util.Locale
 fun VideoSummarizationScreen(viewModel: VideoSummarizationViewModel = hiltViewModel()) {
 
     val context = LocalContext.current
-    val videoList = VideoList().videos
-    var selectedVideoUri by remember { mutableStateOf<Uri?>(videoList.first().uri) }
+    var selectedVideoUri by remember { mutableStateOf<Uri?>(sampleVideoList.first().uri) }
     var isDropdownExpanded by remember { mutableStateOf(false) }
-    var newVideoUrl by remember { mutableStateOf("") }
-    val outputText by viewModel.outputText.collectAsState()
+    val outputTextState  =  viewModel.outputText.collectAsState()
+    val showListenButton by remember {
+        derivedStateOf {
+            outputTextState.value is OutputTextState.Success
+        }
+    }
     var textForSpeech by remember { mutableStateOf("") }
     var textToSpeech: TextToSpeech? by remember { mutableStateOf(null) }
     var isInitialized by remember { mutableStateOf(false) }
     var isSpeaking by remember { mutableStateOf(false) }
     var isPaused by remember { mutableStateOf(false) }
 
-    val videoOptions = videoList.map { it.title to it.uri }
+    val videoOptions = sampleVideoList.map { it.title to it.uri }
```

Comment:
This never changes so should just be extracted from the composable to save it being recomputed everytime.

Feel free to skip because it's not in your PR, but I think an even better change would be that VideoSelectionDropdown should just take in `VideoItem` rather than a `Pair`